cli/src/types.rs

```diff
@@ -124,6 +125,12 @@ pub struct BundleMeta {
     pub codeowners: Option<CodeOwners>,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct WithFilePath<T> {
+    pub file_path: String,
+    pub wrapped: T,
+}
```

Comment:
I think you probably don't need a named struct, but rather just a simple tuple:
https://doc.rust-lang.org/rust-by-example/primitives/tuples.html